### PR TITLE
ci: fix kokoro windows builds?

### PR DESCRIPTION
**Pull request by dbolduc**
credit to @devbww for diagnosing the problem and @coryan for writing the code to copy-paste. (assuming this fix works)

Hopefully this breaks our windows builds...

---
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/8197)

### CODE REVIEW

File: ci/kokoro/windows/build-32.bat [hunk 1]
```diff
@@ -16,18 +16,18 @@ REM Install Bazelisk.
...
 
-REM Configure the environment to use MSVC 2019 and then switch to PowerShell.
+REM Configure the environment to use MSVC %MSVC_VERSION% and then switch to PowerShell.
 call "c:\Program Files (x86)\Microsoft Visual Studio\%MSVC_VERSION%\Community\VC\Auxiliary\Build\vcvars32.bat"
 
 REM The remaining of the build script is implemented in PowerShell.
 @echo %date% %time%
 powershell -exec bypass ci\kokoro\windows\build.ps1
-if %errorlevel% neq 0 exit /b %errorlevel%
+if %ERRORLEVEL% neq 0 exit /b 1
```
Comment (coryan) [inline ci/kokoro/windows/build-32.bat:30]: Maybe consistently use `@if` as above?

File: ci/kokoro/windows/build.bat [hunk 2]
```diff
@@ -27,7 +27,7 @@ call "c:\Program Files (x86)\Microsoft Visual Studio\%MSVC_VERSION%\Community\VC
 REM The remaining of the build script is implemented in PowerShell.
 @echo %date% %time%
 powershell -exec bypass ci\kokoro\windows\build.ps1
-if NOT ERRORLEVEL 0 exit /b 1
+if %ERRORLEVEL% neq 0 exit /b 1
```
Comment (coryan) [inline ci/kokoro/windows/build.bat:30]: same thing here.

**Review by coryan — CHANGES_REQUESTED** [review]
Excellent!  Let's wait until the Windows builds are fixed before merging this?

File: ci/kokoro/windows/build-32.bat [hunk 3]
```diff
@@ -16,18 +16,18 @@ REM Install Bazelisk.
 @echo %date% %time%
 @cd github\google-cloud-cpp
 @powershell -exec bypass ci\kokoro\windows\install-bazelisk.ps1
-@if NOT ERRORLEVEL 0 exit /b 1
+@if %ERRORLEVEL% neq 0 exit /b 1
```
Comment (devbww) [inline ci/kokoro/windows/build-32.bat:19]: I would suggest sticking with `@if ERRORLEVEL 1` (et seq).  `%ERRORLEVEL%` first looks for an environment variable of that name, which, if it exists, is probably not the error level of the last command (https://devblogs.microsoft.com/oldnewthing/20080926-00/?p=20743).

File: ci/kokoro/windows/build-32.bat (hunk 1 above)
Comment (dbolduc) [reply ci/kokoro/windows/build-32.bat]: Done.

File: ci/kokoro/windows/build.bat (hunk 2 above)
Comment (dbolduc) [reply ci/kokoro/windows/build.bat]: Done.

File: ci/kokoro/windows/build-32.bat (hunk 3 above)
Comment (dbolduc) [reply ci/kokoro/windows/build-32.bat]: TIL. Done.

**Review by devbww — APPROVED** [review]

**Pull request merged**